pkg/descheduler/framework/plugins/loadaware/utilization_util.go

```diff
@@ -243,40 +320,97 @@ func evictPodsFromSourceNodes(
 	continueEviction continueEvictionCond,
 	evictionReasonGenerator evictionReasonGeneratorFn,
 ) {
-	var targetNodes []*corev1.Node
-	totalAvailableUsages := map[corev1.ResourceName]*resource.Quantity{}
-	for _, destinationNode := range destinationNodes {
-		targetNodes = append(targetNodes, destinationNode.node)
-
-		for _, resourceName := range resourceNames {
-			quantity, ok := totalAvailableUsages[resourceName]
-			if !ok {
-				switch resourceName {
-				case corev1.ResourceCPU:
-					quantity = resource.NewMilliQuantity(0, resource.DecimalSI)
-				case corev1.ResourceMemory, corev1.ResourceEphemeralStorage, corev1.ResourceStorage:
-					quantity = resource.NewQuantity(0, resource.BinarySI)
-				default:
-					quantity = resource.NewQuantity(0, resource.DecimalSI)
-				}
-				totalAvailableUsages[resourceName] = quantity
+	totalAvailableUsages, targetNodes := targetAvailableUsage(destinationNodes, resourceNames, false)
+	prodAvailableUsages, prodTargetNodes := targetAvailableUsage(prodDestinationNodes, resourceNames, true)
+	bothTotalAvailableUsage, bothTotalNodes := targetAvailableUsage(bothDestinationNodes, resourceNames, false)
+	prodBothAvailableUsage, prodBothTotalNodes := targetAvailableUsage(bothDestinationNodes, resourceNames, true)
```

Comment:
We need to keep in mind that actually `bothTotalNodes` are same with `prodBothTotalNodes`.

Reply:
ok